docs/USING_ADVANCED.md

```diff
@@ -1,3 +1,23 @@
+## Marked instance
+
+By default marked stores options and extensions in the global scope. In node js when you import marked into two scripts and change the options in one script the options will be changed in the other script.
+
+You can create an instance of marked to keep options and extensions locally scoped.
+
+```js
+import { Marked } from 'marked';
+const marked = new Marked([...useArgs]);
```

Comment:
Is this supposed to be an array? The tests make it seems like its an object, not array.

Reply:
The constructor can take any number of options objects just like the use function. The square brackets mean it is optional. I know it is a weird way of writing that but that is how we indicate optional arguments in other parts of the docs.

Reply:
I updated it to be more informative